unittests/AST/DiagnosticConsumerTests.cpp

```diff
@@ -63,12 +63,15 @@ TEST(FileSpecificDiagnosticConsumer, SubConsumersFinishInOrder) {
   auto consumerUnaffiliated = llvm::make_unique<ExpectationDiagnosticConsumer>(
       consumerA.get(), None);
 
-  SmallVector<FileSpecificDiagnosticConsumer::ConsumerPair, 2> consumers;
-  consumers.emplace_back("A", std::move(consumerA));
-  consumers.emplace_back("", std::move(consumerUnaffiliated));
+  SmallVector<FileSpecificDiagnosticConsumer::Subconsumer, 2> consumers;
+  consumers.emplace_back(
+      FileSpecificDiagnosticConsumer::Subconsumer("A", std::move(consumerA)));
+  consumers.emplace_back(FileSpecificDiagnosticConsumer::Subconsumer(
```

Comment:
More `emplace_back`s that shouldn't need to change (all throughout the file).

Reply:
Thanks. Fixed.